six/modules/c++/cphd/unittests/convert_xml_to_string.py

```diff
@@ -0,0 +1,21 @@
+import os, sys
```

Comment:
What does this do?

Reply:
Was supposed to delete this. Just a script I wrote because I was too lazy to format the xml as a string in two of the unittests.